crates/iota-types/src/dynamic_field.rs

```diff
@@ -65,12 +65,11 @@ where
 }
 
 #[serde_as]
-#[derive(Clone, Serialize, Deserialize, JsonSchema, Debug)]
+#[derive(Clone, Serialize, Deserialize, Debug)]
 #[serde(rename_all = "camelCase")]
 pub struct DynamicFieldInfo {
     pub name: DynamicFieldName,
-    #[schemars(with = "Base58")]
```

Comment:
Where was this previously serialized into? Don't we have to worry about deserializing previously serialized base58 values?

Reply:
The `DynamicFieldInfo` of `iota-json-rpc-types` makes sure that deserialization will be done using `Base64`, even if the input value was `Base58`. After that, we are sure that there is only `Base64` in the system. So removing here `#[schemars(with = "Base58")]` and `#[serde_as(as = "Readable<Base58, _>")]` for the `iota-types` type should be safe.

Reply:
discussed privately, agreed to move forward as is